client/components/theme-tier/constants.js

```diff
@@ -17,6 +17,7 @@ import { translate } from 'i18n-calypso';
  * @property {string} label The translated label of the theme tier.
  * @property {string} minimumUpsellPlan The minimum plan required to activate a theme belonging to the tier. Used for upselling purposes.
  */
+// Once if the tier URL path thingy is resolved we could remove this file.
```

Comment:
Wdyt about adding a link to an issue/conversation that clarifies what the thingy is? I fear that otherwise we'll end up forgetting what this means 😅

Reply:
haha right